couchbase/src/main/scala/akka/stream/alpakka/couchbase/model.scala

```diff
@@ -126,7 +129,7 @@ object CouchbaseSessionSettings {
    * Scala API:
    */
   def apply(username: String, password: String): CouchbaseSessionSettings =
-    new CouchbaseSessionSettings(username, password, Nil, None)
+    new CouchbaseSessionSettings(username, password, Nil, None, EnrichAsync)
```

Comment:
Why not just `None, Future.successful)`?

Reply:
That's better, thank you.